packages/sync/src/modules/class-terms.php

```diff
@@ -53,7 +45,7 @@ public function id_field() {
 	 * @return string
 	 */
 	public function table_name() {
-		return 'terms';
+		return 'term_taxonomy';
```

Comment:
Is this expected?

Reply:
we calculate the `WHERE`  against columns in this table.

Reply:
And we're sure this has no impact?